kong/init.lua

```diff
@@ -749,7 +754,7 @@ end
 
 function Kong.ssl_certificate()
   -- Note: ctx here is for a connection (not for a single request)
-  local ctx = ngx.ctx
+  local ctx = get_ctx_table(fetch_table("ctx", 0, 50))
```

Comment:
`50` is a magic number, could we change it to a constant, then we can adjust it in the future?